configs/katello/4.9.yaml

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
this file is missing `:ignores` and `:code_name`, this won't interfere with branching, but something that will be required for when katello actually does the release prep and generates changelog etc